src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -454,6 +468,11 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
             args.insert({DNNL_ARG_DST, output.get_onednn_memory(_pd.dnnl::primitive_desc_base::dst_desc(0), offset)});
         }
 
+        if (_scratchpad_md.get_size() != 0) {
+            auto scratchpad = const_cast<cldnn::memory*>(instance.get_intermediates_memories()[0].get());
```

Comment:
I think `get_onednn_memory` can be marked as `const` instead to avoid this const cast

Reply:
`get_onednn_memory` is marked as `const`.